app/jobs/sms_account_reset_cancellation_notifier_job.rb

```diff
@@ -0,0 +1,13 @@
+class SmsAccountResetCancellationNotifierJob < ApplicationJob
+  queue_as :sms
+
+  def perform(phone:)
+    TwilioService::Utils.new.send_sms(
+      to: phone,
+      body: I18n.t(
+        'jobs.sms_account_reset_cancel_job.message',
+        app: APP_NAME
+      )
+    )
+  end
+end
```

Comment:
I just remembered that both this and `SmsAccountResetNotifierJob` need to be using the same logic as `SmsOtpSenderJob` to determine whether to send the SMS via Programmable SMS or via Verify, and they both need to rescue Twilio errors and log them to analytics, similar to what we do in `TwoFactorAuthenticationController`.

Reply:
Three sms jobs are still using the old format.  What is the implications of them not using this new logic?

Reply:
The implication is that International users might not receive these messages. The reason we purchased the Verify service is to improve deliverability internationally.

Relatedly, would it be be worth it to mention in the email that they should also expect to receive an SMS on their phone (if they have one)?

Reply:
That email idea might be useful.  It would also be helpful for them to get an SMS telling them when they have the email that is used to delete their account.  It seems some users are forgetting to check their email and then the link expires.

Reply:
Steve and I talked about this privately, so I'm documenting here for future reference. I had forgotten that Verify is only for sending OTP codes using Verify's own message. Its use case is not for sending custom messages, so we'll stick with Programmable SMS for this feature.